addons/html_editor/static/src/main/list/list_plugin.js

```diff
@@ -840,6 +841,30 @@ export class ListPlugin extends Plugin {
         return true;
     }
 
+    insertListWithinPre(node) {
+        const listItems = node.querySelectorAll("li:not(.oe-nested)");
+        const getListNestingLvl = (node) => ancestors(node).filter(isListElement).length - 1;
```

Comment:
That's nitpicking, but I'm not sure what's the point of this inner function considering it's only used once. I guess it has the benefit of explaining what level the `lvl` variable is referring to, but if you rename the `lvl` variable to `nestingLvl` then the inner function really adds no benefit anymore... ^^'